qemu/tests/cfg/virtio_fs_support_win_fs.cfg

```diff
@@ -41,6 +41,8 @@
         install_winfsp_path = 'C:\Program Files (x86)'
     autoIt_path = ":\AutoIt3_%PROCESSOR_ARCHITECTURE%.exe"
     script_path = ":\simulate_right_click_on_7z.au3"
+    Win2025:
+        script_path = ":\simulate_right_click_on_7z_latest.au3"
```

Comment:
It might be that the following new windows os will also has the same behavior. So how about set this one as a default one, while for win10,win2016,win2019,win2022 use the old way.